AlbumsNetCoreNoFramework/Program.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Net;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+var http = new HttpListener();
+
+var host = "127.0.0.1";
+var port = 5000;
+var contentType = "application/json";
+
+http.Prefixes.Add($"http://{host}:{port}/");
+http.Start();
+
+var myAlbums = new[]
+{
+    new Album("1", "Pawn Hearts", "Van der Graaf Generator", 26.99),
+    new Album("2", "A Passion Play", "Jethro Tull", 17.99),
+    new Album("3", "Tales from Topographic Oceans", "Yes", 32.99)
+};
+
+// listen
+while(http.IsListening)
+{
+    var context = await http.GetContextAsync();
```

Comment:
A couple of things and it's why this model is so hard to use:
- This needs to be disposed to handle the next request.
- It doesn't handle concurrent requests. You need to dispatch to get concurrency here.

Reply:
+1

There are tradeoffs.  Along the lines of doing it manual, I added manual concurrency. :)